src/migrations/20220331085057-add-api-link-table.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS api_token_project
+        (
+            secret     text NOT NULL,
+            project    text NOT NULL,
+            FOREIGN KEY (secret) REFERENCES api_tokens (secret) ON DELETE CASCADE,
+            FOREIGN KEY (project) REFERENCES projects(id) ON DELETE CASCADE
```

Comment:
will this automatically remove the project from the token if the project is removed? 
If yes: Awesome!

Reply:
Yeah it will!

Reply:
Cascade does rule :)